pkg/kubelet/pluginmanager/cache/actual_state_of_world.go

```diff
@@ -75,6 +75,12 @@ type actualStateOfWorld struct {
 
 var _ ActualStateOfWorld = &actualStateOfWorld{}
 
+// HandlerWithName holds information for handler, infoResp.Name
```

Comment:
Could you share more about the decision to introduce a new data structure? Out of curiosity, where there ways to accomplish the goal without introducing a new data structure?

Reply:
In order to perform unregistration, we need to retrieve plugin handler via socket path.
Existing data structure is keyed by infoResp.Type

Hence I introduced a new struct.

I am open to suggestion on alternate approach.